src/main/java/org/elasticsearch/rest/action/cat/RestPluginsAction.java

```diff
@@ -77,6 +77,7 @@ public RestResponse buildResponse(final NodesInfoResponse nodesInfoResponse) thr
     Table getTableWithHeader(final RestRequest request) {
         Table table = new Table();
         table.startHeaders();
+        table.addCell("id", "alias:i;default:false;desc:unique node id");
```

Comment:
maybe we want to drop the `i` alias here and be consistent with the other cat apis? `id` seems short enough already no?